app/controllers/cuisine_controller.rb

```diff
@@ -0,0 +1,69 @@
+class CuisineController < ApplicationController
+
+  before_action :find_related_restaurants, :only => [:delete, :destroy]
+
+  def index
+    @cuisines = Cuisine.sorted
+  end
+
+  def show
+    @cuisine = Cuisine.find(params.require(:id))
+  end
+
+  def new
+    @cuisine = Cuisine.new
+  end
+
+  def create
+    @cuisine = Cuisine.new(cuisine_params)
```

Comment:
Instead of Cuisine.new and Cuisine.Save you can use Cuisine.create!

create is just new + save together, adding a `!` at the end means that any exception will not be "eaten up" but rather will be thrown immediately

Reply:
Cool, I wasn't familiar with create!
Changed.